prov/efa/src/efa.h

```diff
@@ -635,11 +635,16 @@ struct rdm_peer *rxr_ep_get_peer(struct rxr_ep *ep, fi_addr_t addr)
 	return av_entry->conn.ep_addr ? &av_entry->conn.rdm_peer : NULL;
 }
 
-static inline bool efa_ep_is_cuda_mr(struct efa_mr *efa_mr)
+static inline bool efa_ep_is_hmem_mr(struct efa_mr *efa_mr)
 {
 	return efa_mr ? (efa_mr->peer.iface == FI_HMEM_CUDA): false;
```

Comment:
Considering the upcoming introduction of NEURON, should the condition be `efa_mr->peer.iface != FI_MEM_SYS)`?

Reply:
I'd rather be explicit and only return true if it's an HMEM device that EFA supports.

Reply:
Then how do we handle neuron memory for this function?

Reply:
I've got another PR coming soon that adds it.

https://github.com/rwespetal/libfabric/commit/167907bd4444141b97c95e53e34d38951a0024ce#diff-7888c79bd531a7d7b04b59ebebbf11cc0cc3c29096b5c6bb532d244284933ff0R641

Reply:
I see. Thanks!